docs/install/install.rst

```diff
@@ -0,0 +1,85 @@
+.. meta::
+  :description: Install ROCm Validation Suite
+  :keywords: install, rocm validation suite, rvs, RVS, AMD, ROCm
+
+*****************
+Installing RVS
```

Comment:
This topic (install.rst) is not available on the landing page. 
However, I can see it here:
https://advanced-micro-devices-demo--742.com.readthedocs.build/projects/ROCmValidationSuite/en/742/install/install.html
Should it even be here? Or is this content covered by (install.rst)

Reply:
It is available on the landing page (assume index.rst). Please confirm. @lpaoletti 

<img width="725" alt="rvsinstall" src="https://github.com/ROCm/ROCmValidationSuite/assets/56051583/e2da7cb0-47fd-4c7e-878a-083ceed8972b">

Reply:
Confirmed